containerized_ml_models/model_api_14/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM continuumio/anaconda3
+
+COPY model.pkl api.py requirements.txt /
```

Comment:
I would put ''./' instead of "/" to make sure that you copy these files into CWD. CWD could be changed in the base image. I don't think it's a mistake, if you prefer "/" to "./" leave it like that.